main.go

```diff
@@ -128,7 +137,10 @@ func main() {
 	setupLog = ctrl.Log
 	setupLog.Info("Started with configuration", "configuration", Config)
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	r, err := ctrl.GetConfig()
+	r.RateLimiter = flowcontrol.NewTokenBucketRateLimiter(100, 1000)
+	mgr, err := ctrl.NewManager(r, ctrl.Options{
+		//mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
```

Comment:
actually, merged too quickly - I missed this commented line. @ydayagi pls remove it.